tools/tools.go

```diff
@@ -202,7 +201,22 @@ func EnvmanAddEnvs(envstorePth string, envsList []envmanModels.EnvironmentItemMo
 			sensitive = *opts.IsSensitive
 		}
 
-		if err := EnvmanAdd(envstorePth, key, value, isExpand, skipIfEmpty, sensitive); err != nil {
+		if err := envman.AddEnv(envstorePth, key, value, isExpand, false, skipIfEmpty, sensitive); err != nil {
+			isEnvVarLimitErr := false
+			var envVarTooBigErr envman.EnvVarValueTooLargeError
```

Comment:
It could maybe simplifified
```suggestion
			var envVarTooBigErr envman.EnvVarValueTooLargeError
                        var envVarListTooLargeErr envman.EnvVarListTooLargeError
                        if errors.As(err, &envVarTooBigErr) || errors.As(err, &envVarListTooLargeErr) {
                          return fmt.Errorf("%w.\nTo increase env var limits please visit: %s", err, envVarLimitErrorKnowledgeBaseURL)
                         }
                         return nil
```